pathnames.h

```diff
@@ -64,29 +72,29 @@
  * readable by anyone except the user him/herself, though this does not
  * contain anything particularly secret.
  */
-#define _PATH_SSH_USER_HOSTFILE		"~/" _PATH_SSH_USER_DIR "/known_hosts"
+#define _PATH_SSH_USER_HOSTFILE     "~" _PATH_SEPARATOR _PATH_SSH_USER_DIR _PATH_SEPARATOR "known_hosts"
```

Comment:
Previously, a tab character was used here to align the text, but this did not work consistently across the editor and the terminal (the following line did not align in `git diff`). I replaced it with spaces so that it is consistent across viewing environments.